versions.yaml

```diff
@@ -82,14 +82,17 @@ module-sets:
       - go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful/test
       - go.opentelemetry.io/contrib/zpages
   experimental-metrics:
-    version: v0.28.0
+    version: v0.29.0
+    modules:
+      - go.opentelemetry.io/contrib/instrumentation/host
+      - go.opentelemetry.io/contrib/instrumentation/host/example
+      - go.opentelemetry.io/contrib/instrumentation/runtime
+      - go.opentelemetry.io/contrib/instrumentation/runtime/example
+  experimental-metric-exporters:
+    version: v0.29.0
```

Comment:
I think we should deprecate all of these exporters.  `dogstatsd` can only support counters and gauges and `cortex` is probably better served by shipping OTLP to a collector and using its PRW exporter.  Deprecating them now will allow us to excise them by the time we start landing significant metrics API/SDK changes.

Reply:
Sure, will do.